### PR TITLE
doc improvements

### PR DESCRIPTION
**Pull request by scopatz**
to make the subprocess distinctions a little clearer hopefully

### CODE REVIEW

**Comment by scopatz** [comment]
This does not need a news entry

File: docs/subproc_types.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.. _subproc_types:
+
+***********************
+Subprocess Types Tables
+***********************
+Xonsh has a few different ways to launch subprocesses, each with their own
+unique interface depending on your need. The following table is a quick reference
+for the different suprocesses. The collumns have the following meaning:
```
Comment (gforsyth) [inline docs/subproc_types.rst]: *columns

File: docs/subproc_types.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+:Type: The sytnax for an example subprocess ``cmd``.
```
Comment (gforsyth) [inline docs/subproc_types.rst]: *syntax

File: docs/subproc_types.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
+.. _subproc_types:
+
+***********************
+Subprocess Types Tables
```
Comment (gforsyth) [inline docs/subproc_types.rst]: Is it "Subprocess Types Tables" or the "Subprocess Types Table"?

**Comment by scopatz** [comment]
Thanks @gforsyth! Here are the typo fixes

**Comment by scopatz** [comment]
ping!

**Pull request merged**